metadata-ingestion/src/datahub/ingestion/source/state/stale_entity_removal_handler.py

```diff
@@ -194,7 +194,7 @@ def compute_job_id(cls, platform: Optional[str]) -> JobId:
         return JobId(f"{platform}_{job_name_suffix}" if platform else job_name_suffix)
 
     def _init_job_id(self) -> JobId:
-        platform: Optional[str] = getattr(self.source, "platform")
+        platform: Optional[str] = getattr(self.source, "platform", "default")
```

Comment:
@treff7es : wanted to bring this diff to your attention since it is related to stateful ingestion. Previously this handler was expecting the class to have a `platform` member variable and failing hard if it did not.